distributions/distribution-resources/src/main/resources/conf/html/readme.txt

```diff
@@ -0,0 +1,2 @@
+Check out the openHAB2 documentation for more details:
+http://docs.openhab.org
```

Comment:
See https://github.com/openhab/openhab2-addons/issues/311.
In short: Files that you put in this folder will be available on the HTTP server of openHAB at /static/.